ruff.toml

```diff
@@ -0,0 +1,47 @@
+# MAX ACCEPTABLE LINE LENGTH
+line-length = 88
+
+
+#CACHE DIR FOLDER FOR RUFF
+cache-dir = "~/.cache/ruff"
+
+
+#EXCLUDED FILE TYPES
+exclude = [".git", "__pycache__", "data", "tools", "env", "venv"]
```

Comment:
Let's use `extend-exclude` instead, so we benefit from Ruff's default exclude list.